Test/FMI3/fmi3_import_error_handling_model_variables.cpp

```diff
@@ -143,12 +140,12 @@ TEST_CASE("Test missing required attribute plus other attribute errors/warnings"
 
     REQUIRE(fmi3_testutil_log_contains(tfmu, "Parsing XML element 'Float64': required attribute 'name' not found"));
 
-    // We do not check attribute errors if name/valueReference are erroneous
+    // We do not check attribute errors if name/valueReference are erroneous; test absence of error message
     REQUIRE(!fmi3_testutil_log_contains(tfmu, "A variable must not refer to itself in the attribute 'previous'."));
 
     REQUIRE(fmi3_testutil_log_contains(tfmu, "Fatal failure in parsing ModelVariables. Variable(s) failed to parse or an essential error check failed.")); // counts as 2
 
-    REQUIRE(fmi3_testutil_get_num_problems(tfmu) == 4);
+    REQUIRE(fmi3_testutil_get_num_problems(tfmu) == 3);
```

Comment:
The return value changes in fmi3_xml_variable.c.

Zero (rather than negative) returns would trigger the verbose buffer cleanup of non-parsed attributes. I.e., `Attribute '%s' not processed by element '%s' handle"`, where I believe the intention is to identify attributes that may be misplaced for a variable.